content/en/contribute/_index.md

```diff
@@ -20,4 +20,17 @@ The [Development](https://forum.communityhealthtoolkit.org/c/developement/7) sec
 - [Share your expertise](https://forum.communityhealthtoolkit.org) on our Forum
 - Sign up for our [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates
 
+**Are you a partner wondering how issues are prioritized?**
+
+There are a number of factors that contribute to prioritization of work. The following represents important factors that are often evaluated together to have the best change for the largest impact if a partner will: 
```

Comment:
I think this line was from me in the first place, so here's my correction of my own error:
```suggestion
There are a number of factors that contribute to prioritization of work. The following represents important factors that are often evaluated together to have the best chance for the largest impact if a partner will: 
```